### PR TITLE
fix(client): fixed icon for the general part of settings page broken on firefox

### PR DESCRIPTION
**Pull request by CodeAnk2829**
Close: #7179

## PR Details
Normal looking `cog` as icon for the general tab was broken  across both the `service` and `project` settings pages on firefox while it was looking fine on different browsers. This PR fixes the icon for the general part of settings on firefox.

### Summary
🖼️🎨🚀

Adjust inner tab link icon width to match design and improve layout. This pull request updates the CSS file `InnerTabLink.scss` and other files related to the inner tabs component.

> _`inner tab` icon_
> _width adjusted to match design_
> _autumn leaves align_

### Walkthrough
* Set the inner tab link icon width to 22px to match the design and align with the text ([link](https://github.com/amplication/amplication/pull/7188/files?diff=unified&w=0#diff-54325424dcb60e91019a0f5857c98d3bc06f457203ac5d986ca0da9a0fa6d5ecR21))

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

image

### CODE REVIEW

File: packages/amplication-client/src/Layout/InnerTabLink.scss [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
   .amp-icon {
     font-size: var(--title3-font-size);
+    width: 22px;
```
Comment (yuval-hazaz) [inline packages/amplication-client/src/Layout/InnerTabLink.scss:21]: We recently aligned all CSS properties on the design-system, and you should not manually change the CSS of the icon.
Please try to remove the width and font-size properties from the CSS - and instead use the size property on the Icon component.

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]

File: packages/amplication-client/src/Layout/InnerTabLink.scss (hunk 1 above)
Comment (CodeAnk2829) [reply packages/amplication-client/src/Layout/InnerTabLink.scss:21]: Thanks for reviewing @yuval-hazaz, started working on it.

**Review by CodeAnk2829 — COMMENTED** [review]
Hey @yuval-hazaz , I have changed the font-size property to width in amp-icon class and tried to change the font-size on icon components accordingly. It still resolved the same issue. Please consider the last commit. And if something I would have done wrong again then please let me know, I will try my best to resolve that as well.

**Comment by yuval-hazaz** [comment]
@CodeAnk2829 thank you very much for your efforts!
I decided to dig into this issue a little more to find the root cause. 
It was actually about "white-space:nowrap" that helped fix the issue. somehow there was not enough space for the icon of FF and it decided to break the icon to two lines.

Other than that I prevented your changes, and changed the size of the icon from medium to small (instead of using a local style for font-size).

I appreciate your contribution. we will merge this OR and your commits will count! thank you

**Review by yuval-hazaz — APPROVED** [review]

**Comment by CodeAnk2829** [comment]
Thank you so much for your appreciation @yuval-hazaz and @souravjain540. I loved to contribute in this awesome project and I will keep contributing in [amplication](https://amplication.com/). This was my first contribution, regularly checking the issue that suits me so that I could start working on it.

**Pull request merged**